code/modules/recycling/conveyor_assembly.dm

```diff
@@ -55,9 +55,13 @@
 	placeimage.overlays += arrow
 
 /obj/item/stack/conveyor_assembly/drag_success(mob/user, turf/T)
-	if(user.Adjacent(placeloc) && use(1))
-		new /obj/structure/conveyor_assembly(placeloc, placeimage.dir)
-	if(amount)
+	var/direction = get_dir(placeloc, T)
```

Comment:
While I don't think this is necessary since MouseDrag should have been called at least once before this I don't want any rare phantom bugs occurring